scripts/rspec/rspec.ps1

```diff
@@ -322,13 +327,20 @@ function ReplaceTokens($text) {
 
 ### SCRIPT START ###
 
+$sonarpediaFolder = $sonarpediaMap.Get_Item($language)
+Write-Host "Will change directory to $sonarpediaFolder to run rule-api"
+cd $sonarpediaFolder
```

Comment:
you could use `pushd $sonarpediafolder` and then `popd` instead of `cd ../../..`

Reply:
also, would it be possible to do the same without changing folders?